lib/tlsca/ca.go

```diff
@@ -150,39 +150,39 @@ type RouteToApp struct {
 	// SubjectPublicKeyInfo like the CA pin is for UX consistency. For example,
 	// the SessionID is emitted in the audit log, using a UUID matches how SSH
 	// sessions are identified.
-	SessionID string
+	SessionID string `json:"session_id"`
```

Comment:
Same here - I would be very careful not presenting internal objects to the users. Soon as we do that, they become a public API which we won't be able to break.